package.json

```diff
@@ -38,12 +38,16 @@
     "imagely": "imagely.bin.js"
   },
   "dependencies": {
-    "bluebird": "^2.10.2",
-    "image-size": "^0.3.5",
-    "lodash": "^3.10.1",
-    "phantom": "^0.8.0",
-    "phantomjs": "^1.9.18",
-    "request-promise": "^0.4.3",
-    "yargs": "^3.26.0"
+    "bluebird": "^3.5.0",
+    "image-size": "^0.5.1",
+    "imagely": "imagely-v0.1.12.tgz",
```

Comment:
I mistakingly added that, but took it out in a later PR.

How I was trying to test imagely locally as a package using `yarn pack`, http://podefr.tumblr.com/post/30488475488/locally-test-your-npm-modules-without-publishing